spec/supabase_js_v2.yml

```diff
@@ -494,6 +505,8 @@ functions:
     $ref: '@supabase/gotrue-js.GoTrueClient.reauthenticate'
     notes: |
       - This method is used together with `updateUser()` when a user's password needs to be updated.
+      - If you require your user to reauthenticate before updating their password, you need to enable the **Secure password change** option in your [project's email provider settings](/project/_/auth/providers).
```

Comment:
should this be `project/_/auth/providers`? Perhaps `https://app.supabase.com/project/_/auth/providers`?